src/wwwroot/template-editor/adapterService.js

```diff
@@ -26,9 +30,30 @@
       getTiendaNubeProducts: getTiendaNubeProducts,
       overrideUploadFiles: overrideUploadFiles
     };
+    
+    var langKey = null;
+    init();
 
     return service;
 
+    function init() {
+      console.log("test");
+      langKey = getPreferredLanguage();
+    };
+
+    /**
+     * Return a language identifier value stored previously in the local storage of the browser
+     * @example
+     * ```js
+     * // Return 'en' or 'es'
+     * var langKey = getPreferredLanguage();
+     * ```
+     */
```

Comment:
In my opinion, this documentation is not necessary, this code in enough clear by his own, and also it is a private function.